tests/test_modeling_tf_common.py

```diff
@@ -184,7 +184,7 @@ def test_saved_model_creation(self):
 
         with tempfile.TemporaryDirectory() as tmpdirname:
             model.save_pretrained(tmpdirname, saved_model=True)
-            saved_model_dir = os.path.join(tmpdirname, "saved_model")
+            saved_model_dir = os.path.join(tmpdirname, "saved_model", "1")
```

Comment:
Out of curiosity, why is this change necessary?

Reply:
To be aligned with all the tests that are about saved model. And also because the files are actually created inside `tmpdirname/saved_model/1` and not in `tmpdirname/saved_model`.